tech/languages/csharp/dotnet-installation.md

```diff
@@ -17,21 +17,29 @@ $ sudo dnf copr enable @dotnet-sig/dotnet
 
 ### .NET Core SDK
 
-.NET Core SDK will enable building and publishing C# source code.
+.NET Core SDK enables building and publishing of C# source code.
+
+To install the latest SDK version:
+
+```
+$ sudo dnf install dotnet
+```
+
+To install the latest SDK for a specific .NET Core Runtime, where the _x_ stands for major, _y_ for minor version of the runtime:
 ```
-$ sudo dnf install dotnet-sdk-2.0
+$ sudo dnf install dotnet-sdk-x.y
 ```
 
 ### .NET Core Runtime only
 
-You can install runtime only, if you are just deploying already prebuilt application.
+To install runtime only, for example to merely deploy already prebuilt applications, where _x_ stands for major and _y_ stands for minor version:
 ```
-$ sudo dnf install dotnet-runtime-2.0
+$ sudo dnf install dotnet-runtime-x.y
 ```
 
 ## Microsoft .NET Core packages
```

Comment:
Is there a good reason to have this section? The Java page has no links to non-Fedora binaries. The Haskell page has no links to non-Fedora binaries. Same for python.

Reply:
I don't know, I just always linked to all the resources as a habit...

Reply:
Well if we want to update it, this is a good time to do so :P I am feeling very "passive" about it, so if you tell me to remove it I will...